src/ranges.rs

```diff
@@ -0,0 +1,453 @@
+use endianity::{Endianity, EndianBuf};
```

Comment:
Other files have `#![deny(missing_docs)]` at the top, which seems like a good idea.